kms-message/src/kms_request.c

```diff
@@ -762,6 +762,13 @@ kms_request_validate (kms_request_t *request)
    }
 }
 
+/* append_http_endofline appends an HTTP end-of-line marker: "\r\n". */
+static void
+append_http_endofline (kms_request_str_t *str)
+{
+   kms_request_str_append_chars (str, "\r\n", 2);
+}
+
```

Comment:
To confirm, the reason why a new `append_http_endofline` function added instead of changing `kms_request_str_append_newline` to append `\r\n` is because not all KMS request content is HTTP; is this correct?

Reply:
Correct. KMIP does not use HTTP.